x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/utils.ts

```diff
@@ -305,7 +305,12 @@ export const getInvalidConnectors = async (
   rules: PromiseFromStreams[],
   actionsClient: ActionsClient
 ): Promise<[BulkError[], PromiseFromStreams[]]> => {
-  const actionsFind = await actionsClient.getAll();
```

Comment:
If the user is unauthorized for actions `read`, this will throw a 403 error so we wrapped it in a try-catch and default to an empty result set. I'm open to updating this for another proposed fix.